### PR TITLE
added code to daemonize the process and create a named pipe and start listening to it

### PR DESCRIPTION
**Pull request by mebjas**
Tasks done
 - Added proposal as `PROPOSAL.md` in `containerEngine/vagrant` dir
 - Script takes in a command line argument and `starts`, `stops` or `restarts` the daemon accordingly
 - Script creates a named pipe and start listening to it, for commands

### CODE REVIEW

**Comment by pchaigno** [comment]
@mebjas Did you talk with @northdpole about the name of the folder? Maybe we should make sure now that you and @a0xnirudh are using the same folder... That way we wouldn't have to move everything after :-)

File: containerEngine/vagrant/PROPOSAL.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+##Hackademic Vagrant Sandboxed for Challenges
```
Comment (pchaigno) [inline containerEngine/vagrant/PROPOSAL.md]: I don't think your proposal should be shipped with Hackademic's source code. It's probably a good idea to host it somewhere (maybe in the README of your fork?) though.
@northdpole ?

**Comment by mebjas** [comment]
@pchaigno: @northdpole himself suggested that this should be hosted at `containerEngine/vagrant`.

File: containerEngine/vagrant/PROPOSAL.md (hunk 1 above)
Comment (northdpole) [inline containerEngine/vagrant/PROPOSAL.md]: @pchaigno it's good for transparency, I'll move it to the wiki part once the project wraps up :)

**Comment by pchaigno** [comment]
> @northdpole himself suggested that this should be hosted at containerEngine/vagrant.

Yep, I talked with @northdpole and we think you should merge your work into the branch [`minhaz/vagrant`](https://github.com/Hackademic/hackademic/tree/minhaz/vagrant) instead of `next-dev` for now. You will probably have to reopen the pull request to do so.

File: containerEngine/vagrant/PROPOSAL.md (hunk 1 above)
Comment (pchaigno) [inline containerEngine/vagrant/PROPOSAL.md]: :+1:

**Pull request closed without merging**